backend/src/routes/preferences.js

```diff
@@ -0,0 +1,94 @@
+const express = require('express');
+var mongoose = require('mongoose');
+const Preferences = require("../models/preferencesModel");
+const router = express.Router();
+
+/**
+ * Get Preferences Object from Preferences Collection by Id of respective user
+ * 
+ * Route: GET /api/user/:userId/preferences where userId is the ID of user
+ */
+router.get('/:userId/preferences', async (req, res) => {
```

Comment:
Are we using async/await API then?

Reply:
It helps with error handling I think since it makes it easier through a try-catch. Idk what the consensus is for the best API though.

Reply:
I can change mine to fit this then.